src/models/IDPDiscovery.js

```diff
@@ -50,7 +49,7 @@ function (Okta, PrimaryAuthModel, CookieUtil, Enums, Util) {
       var username = this.settings.transformUsername(this.get('username'), Enums.IDP_DISCOVERY),
           remember = this.get('remember'),
           lastUsername = this.get('lastUsername'),
-          resource = 'acct:' + username,
+          resource = 'okta:acct:' + username,
```

Comment:
does this mean that previous versions of the widget won't work anymore for idp discovery? Since those versions will be sending `acct:username` without the `okta:` prefix

Reply:
https://github.com/okta/okta-core/pull/29042 will support both schemas